dpdbaltics.php

```diff
@@ -534,7 +534,7 @@ public function hookDisplayCarrierExtraContent($params)
             $sameDayDeliveryPresenter = $this->getModuleContainer()->get('invertus.dpdbaltics.presenter.same_day_delivery_message_presenter');
             $return .= $sameDayDeliveryPresenter->getSameDayDeliveryMessageTemplate();
         }
-        $return .= $carrierPhoneService->getCarrierPhoneTemplate($this->context->cart->id);
+        $return .= $carrierPhoneService->getCarrierPhoneTemplate($this->context->cart->id, $carrier->id_reference);
```

Comment:
You pass carrier reference and everywhere else it is carrier ID, this is misleading, as those values has completly different meanings

Reply:
Carrier ID changes constantly, after carrier changes in presrashop, carrier reference is always the same.